Week2/Homework/assets/question-six.js

```diff
@@ -0,0 +1,10 @@
+let nlCities = [];
+console.log('the value of the array is 0');
+console.log(nlCities);
+
+const daanFavAnimals = ['lions', 'lynxes', 'Hawks'];
+console.log(daanFavAnimals);
+
+daanFavAnimals[3] = 'baby pig';
```

Comment:
Ah, yeah this works, but a cleaner (and easier) solutio is to use [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push)